docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| ---                                  | -            | -              | -----         |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
+| Any other qualifier                  | MUST NOT     | -              | -             |
+
+##### 7.1.2.6.10 Extended Key Usage
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST         |
+| `id-kp-clientAuth`      | 1.3.6.1.5.5.7.3.2 | MAY          |
+| `id-kp-codeSigning`     | 1.3.6.1.5.5.7.3.3 | MUST NOT     |
+| `id-kp-emailProtection` | 1.3.6.1.5.5.7.3.4 | MUST NOT     |
+| `id-kp-timeStamping`    | 1.3.6.1.5.5.7.3.8 | MUST NOT     |
+| `id-kp-OCSPSigning`     | 1.3.6.1.5.5.7.3.9 | MUST NOT     |
+| `anyExtendedKeyUsage`   | 2.5.29.37.0       | MUST NOT     |
+| Any other value         | -                 | SHOULD NOT   |
+
+##### 7.1.2.6.11 Key Usage
+
+The acceptable Key Usage values vary based on whether the Certificate's `subjectPublicKeyInfo` identifies an RSA public key or an ECC public key. CAs MUST ensure the Key Usage is appropriate for the Certificate Public Key.
+
+Table: Key Usage for RSA Public Keys
+
+| __Key Usage__      | __Permitted__ | __Required__     |
+| ----               | -             | -                |
+| `digitalSignature` | Y             | SHOULD           |
+| `nonRepudiation`   | N             | --               |
+| `keyEncipherment`  | N             | --               |
+| `dataEncipherment` | N             | --               |
+| `keyAgreement`     | Y             | MAY              |
+| `keyCertSign`      | N             | --               |
+| `cRLSign`          | N             | --               |
+| `encipherOnly`     | N             | --               |
+| `decipherOnly`     | N             | --               |
+
+**Note**: At least one Key Usage MUST be set for RSA Public Keys. The `digitalSignature` bit is REQUIRED for use with modern protocols, such as TLS 1.3, and secure ciphersuites, while the `keyAgreement` bit MAY be asserted to support older protocols, such as TLS 1.2, when using insecure ciphersuites. Subscribers MAY wish to ensure key separation to limit the risk from such legacy protocols, and thus a CA MAY issue a Subscriber certificate that only asserts the `keyAgreement` bit. For most Subscribers, the `digitalSignature` bit is sufficient, while Subscribers that want to mix insecure and secure ciphersuites with the same algorithm MAY choose to assert both `digitalSignature` and `keyAgreement` within the same certificate, although this SHOULD NOT be done by default.
```

Comment:
I'm not sure I follow what you're proposing. This note is trying to convey why you SHOULD include this (and hopefully we will MUST it, if not in this version, then I intend to do so in a follow-up)

That seems like it's not obvious, so there's room for improvement here, but I'm not sure which part you found least clear, to focus on rewording.

Reply:
As I was reading through, I thought that it's actually great you provided some security context for including/not including the `keyAgreement` bit and the support for older protocols. Then I asked myself if we should do something similar (in terms of guidance) for the inclusion of the keyUsage extension or not. I thought it would be nice to add similar language to motivate CAs to add the keyUsage extension. Obviously I would not be against a MUST requirement for end-entity certificates but until then, adding reasons why the KU should exist in end-entity certs would be useful.

Reply:
Since MAY and SHOULD have the same compliance weight (i.e. it's optional under both), I've clarified 7.1.2.6.6 to make this a SHOULD. I'd also be quite supportive of making this a MUST at whatever harmonized date we choose.

Hopefully the context in this section would be sufficient for why the SHOULD (vs MAY) exists, but let me know if you think it needs more. I'm struggling since the SHOULD reasoning why should hopefully be easy to see from this existing note.